README.md

```diff
@@ -39,6 +39,15 @@ Rewrite the files to resolve any style issues with:
 make fmt.write
 ```
 
+## Language Contribution Guide
+
+We would love for you to contribute to different languages and help make it even better than it is today! As a language contributor, here are some steps we would like you to follow:
+
+1. Add a translation to `./src/i18n.js`
+2. Add the language option to `./src/App.js`. Use the format e.g. `French (Français)`
```

Comment:
As per https://github.com/bndw/wifi-card/issues/80#issuecomment-887598899, looks like we should do `latinName - nativeName` instead of `latinName (nativeName)` to support RTL languages.

Let's wait until https://github.com/bndw/wifi-card/pull/149 shakes out to determine.